components/bldc_haptics/include/bldc_haptics.hpp

```diff
@@ -186,25 +186,26 @@ template <MotorConcept M> class BldcHaptics : public BaseComponent {
   /// @return Current position of the haptic motor
   float get_position() const { return current_position_; }
 
+  /// @brief Sets the current position of the haptic motor to
+  ///        within the bounds of the current detent config
+  /// @param position Position to set the haptics to
+  void set_position(const int position) {
+    current_position_ = std::clamp<int>(detent_config_.min_position, detent_config_.max_position, position);
```

Comment:
Your clamp is out of order - it's value, low, high

Reply:
Man I don't understand how I got this wrong, sharp